main.go

```diff
@@ -80,13 +84,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	// TODO: Enable webhooks
-	/*
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
Just mentioning personal preference, not an issue:
For environment variable name, personally I'd prefer an application-specific prefix to avoid collisions. E.g HVPA_ENABLE_WEBHOOKS. This becomes more relevant in the context of naming consistency, after the application grows and suddenly needs some very collision-prone variable, like ENABLE_LOGGING.